oss-cloud-backend/event_mock.json

```diff
@@ -0,0 +1,9 @@
+{

```

Comment:
Can we remove this file?
It's not used except as an example for invoking lambda, right? If that's the case we can remove it.

Reply:
File removed.